dbms/tests/queries/0_stateless/01009_insert_select.sql

```diff
@@ -0,0 +1,23 @@
+Set any_join_distinct_right_table_keys=1;
+DROP TABLE IF EXISTS test.test_insert_t1;
```

Comment:
Do not use `test.` database explicitly in tests.

Reply:
I'll remove it.